estate/models/estate_property.py

```diff
@@ -4,7 +4,7 @@
 
 
 class EstateProperty(models.Model):
-    _name = "estate_property"
+    _name = "estate.property"
```

Comment:
This commit is not a [REF]. It shouldn't exist. You should be able to edit previous commits so that the correct version of this line is in the correct commit